tools/harness-thci/OpenThread_Uno.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python
```

Comment:
`OpenThread_Uno.py`, `uno` is the internal code code, should we use the name for public?